build/dsc.rb

```diff
@@ -11,7 +11,6 @@
 if Bundler.rubygems.find_name('mof').any?
   require 'mof'
   require 'cim'
-  require 'charlock_holmes/string'
```

Comment:
FYI, this is a monkey patch against Ruby `String` which reimplements `force_encoding` - so I would grep the code for that to find any usages just in case.